equinox-shipment/Domain/Container.fs

```diff
@@ -0,0 +1,56 @@
+module Container
+
+type Container = { id: string; shipmentIds: string[]; finalized: bool }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | ContainerCreated    of containerId  : string
+        | ContainerFinalized  of shipmentIds  : string[]
+        | Snapshotted         of Container
+        interface TypeShape.UnionContract.IUnionContract
+
+    let codec = FsCodec.NewtonsoftJson.Codec.Create<Event>()
+
+module Fold =
+
+    type State = Container
+
+    let initial: State = { id = null; shipmentIds = [||]; finalized = false }
```

Comment:
why does this need to track the id ? while that's a very common thing, its not necessary

Reply:
I'll remove it thanks!

Reply:
Removed thanks!